R/pkg/R/functions.R

```diff
@@ -1179,6 +1179,27 @@ setMethod("soundex",
             column(jc)
           })
 
+#' spark_partition_id
```

Comment:
Minor nit: The convention we are using in SparkR is to have a descriptive title for the function. So in this case it would be something like `Return the partition ID as a column`. (There might be other places which need to fixed to match this convention as well -- We discussed this in #13394